flocker/node/agents/functional/test_cinder.py

```diff
@@ -171,6 +171,11 @@ def test_name(self):
                 flocker_volume.blockdevice_id).display_name,
             u"flocker-{}".format(dataset_id))
 
+    @flaky('FLOC-3347')
+    def test_get_device_path_device(self):
+        super(CinderBlockDeviceAPIInterfaceTests,
+              self).test_get_device_path_device()
+
 
```

Comment:
it doesn't bother me personally, but I would assume someone else doing this review would ask for new tests to be added in a separate PR, as this PR is essentially about adding the @flaky decorator functionality.